specs/src/public_key_cryptography.md

```diff
@@ -14,7 +14,7 @@ A highly-optimized library is available in C (<https://github.com/bitcoin-core/s
 
 ### Public-keys
 
-Secp256k1 public keys can be compressed to 257-bits (or 33 bytes) per the format described [here](https://github.com/cosmos/cosmos-sdk/blob/v0.46.15/docs/basics/accounts.md#public-keys).
+Secp256k1 public keys can be compressed to 256-bits (or 33 bytes) per the format described [here](https://github.com/cosmos/cosmos-sdk/blob/v0.46.15/docs/basics/accounts.md#public-keys).
```

Comment:
A compressed Secp256k1 public key is 256 bits for the x coordinate and 1 bit to determine if the y coordinate is even or odd. 256 + 1 = 257 which is 33 bytes.

Reply:
Sorry

Reply:
Sorry

Reply:
no worries